### ISSUE TITLE
Unable to add e621(furry.booru.org) & rule34 (rule34.xxx) to sources

### ISSUE THREAD

**Issue by fakesern** [Bionus/imgbrd-grabber#914]
### What steps will reproduce the problem?

Clicking sources -> Add -> either https://e621.net/ OR furry.booru.org OR rule34.xxx

### What is the expected behavior? What do you get instead?

Expected behavior is to be added to the source list and be able to search different boorus & download. Instead i either get "unable to guess sites type" or it gets added, but when i search its like it is not there.

### How often does this problem occur?

100% of time.

### What version of the program are you using? On what operating system?

5.2.4 on Win10

### Please provide any additional information below

This might just be me being stupid. I have tried setting the sites type to all the different boorus. Any and all help is much appreciated! Sorry to bother and have a nice day :D

**Comment by Bionus** [comment]
If it can't guess a site type, you can just tell him by selecting it from the select box. Plus, e621 and rule34.xxx are included by default.

booru.org is Gelbooru 0.1, e621 is Danbooru and rule34.xxx is Danbooru.

**Comment by fakesern** [comment]
Hey, Thanks for the quick reply, and sorry for being a little slow.
To start of, yes they were included in the start, but I deleted most of the presets cause I thought I would not be needing them.

Now, I got e621.net to work. What i originally did was add the full adrres https://e621.net/ but while looking at the log, i found out that your program automatically adds the http:// and the / at the end. Deleting this extra resolved this issue.

I found out furry.booru.org is just a mirror of e621.net

I got rule34.xxx to work by just re-adding it. It is gelbooru by the way.

So this is just me being stupid. Thanks for creating this program, i hope life goes well for you! Sorry for wasting your time :/
